docs/build-tools-index.md

```diff
@@ -20,6 +20,8 @@ sections below will be filled out.
 
 ## Wallets
 
+See more on Wallets [here](build-wallets.md).
+
 - [Polkadot-JS Apps - Accounts](https://polkadot.js.org/apps/#/accounts)
```

Comment:
Please remove all of these wallets so that there is only one place (the Wallets page) where there is a list of wallets (to prevent there being two sources of truth... just like normalizing a database)